oom-frontend/src/app/shared/modules/user/user-finder/user-finder.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { UserFinderComponent } from './user-finder.component';
+
+describe('UserFinderComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
describe("UserFinderComponent", () => {
```